docs/sphinx/developers/format-documentation.txt

```diff
@@ -0,0 +1,107 @@
+Adding format/reader documentation pages
+========================================
+
+.. _Velocity: http://velocity.apache.org/
+
+Most documentation pages for the supported formats and readers are
+auto-generated. These pages should not be modified directly. This page
+explains how to amend/extend this part of the Bio-Formats documentation.
+
+The :sourcedir:`Bio-Formats testing framework <components/autogen>` component
+contains most of the infrastructure to run automated tests against the data
+repository.
+
+Formats
+-------
+
+After checking out source code and building all the JAR files (see
+:doc:`building-bioformats`), the supported formats pages can be generated
+using the :program:`ant` ``gen-format-pages`` target under the :file:`autogen`
+component::
+
+  $ ant -f components/autogen/build.xml gen-format-pages
+
+This target will read the metadata for each format stored under
+:source:`format-pages.txt <components/autogen/src/format-pages.txt>` and
+generate a reStructuredText file for each format stored under
+:file:`formats/<formatname>.txt` as well as an index page for all supported
+formats using Velocity_.
+
+The :file:`format-pages.txt` is an INI file where each section corresponds to
+a particular format given by the section header. Multiple key/values should be 
+defined for each section:
+
+pagename
+  The name of the output reStructuredText file. If unspecified, the section 
+  header will be used to generate the filename.
+
+extensions
+  The list of extensions supported for the format
+
+owner
+  The owner of the file format
+
+developer
+  The developer of the file format
+
+bsd
+  A `yes/no` flag specifying whether the format readers/writers are under the
+  BSD license
+
+export
+  A `yes/no` flag specifying whether the format can be exported
+
+versions
+  A comma-separated list of all versions supported for this format
+
+weHave
+  A bullet-point list describing the supporting material we have for this
+  format including specification and sample datasets
+
+weWant
+  A bullet-point list describing the supporting material we would like to have
+  for this format
+
+pixelRating
+  See :term:`Ratings legend and definitions`. Available choices are: 
+  `Poor`, `Fair`, `Good`, `Very Good`, `Outstanding`
+
+metadataRating
+  See :term:`Ratings legend and definitions`. Available choices are: 
+  `Poor`, `Fair`, `Good`, `Very Good`, `Outstanding`
+
+opennessRating
+  See :term:`Ratings legend and definitions`. Available choices are: 
+  `Poor`, `Fair`, `Good`, `Very Good`, `Outstanding`
+
+presenceRating
+  See :term:`Ratings legend and definitions`. Available choices are: 
+  `Poor`, `Fair`, `Good`, `Very Good`, `Outstanding`
+
+utilityRating
+  See :term:`Ratings legend and definitions`. Available choices are: 
+  `Poor`, `Fair`, `Good`, `Very Good`, `Outstanding`
+
+reader
+  A string or a comma-separated list of all readers for this format
+
+notes
+  A set of notes to be added
```

Comment:
"Additional relevant information e.g. that we cannot distribute specification documents to third parties." ?

Not hugely fussed, but at least makes it sound a little less self-explanatory ;)